### PR TITLE
return Promise from Android .isSupported() to match iOS usage

### PR DESCRIPTION
**Pull request by nemophrost**

### CODE REVIEW

File: TouchID.android.js [hunk 1]
```diff
@@ -8,6 +8,6 @@
 
 export default {
   isSupported() {
-    return false;
+    return Promise.resolve(false);
```
Comment (eagleeye) [inline TouchID.android.js]: actually it should be reject, see example in README

**Comment by nemophrost** [comment]
Fixed

**Comment by alexgsummer** [comment]
Can we get this PR merged in?

**Comment by zibs** [comment]
@alexgsummer Hey Alex, Android support is on it's way and so I don't really think it's needed to merge this in, as once Android lands the interface will be mostly the same as iOS and so the promise will be resolved/rejected if it is supported or not.

**Comment by alexgsummer** [comment]
@zibs Perfect. Thanks for the heads up!

**Comment by zibs** [comment]
This has been fixed in master now.

**Pull request closed without merging**